### PR TITLE
110: Replace MO CLI with Python API and use PTQ nncf API

### PR DESCRIPTION
**Pull request by aleksandr-mokrov**
1. MO CLI is replaced with Python API
2. POT is replaced with PTQ nncf API.
3. 110-ct-segmentation-quantize.ipynb is removed. Only 110-ct-segmentation-quantize-nncf.ipynb is actual.
4. pytorch-monai-training.ipynb is updated: PyTorch Lightning has new incompatible interface (https://github.com/Lightning-AI/lightning/pull/16520 for example), F1 score metric from POT replaced by BinaryF1Score from torchmetrics.

### CODE REVIEW

File: notebooks/110-ct-segmentation-quantize/110-ct-segmentation-quantize-nncf.ipynb [hunk 1]
```diff
@@ -430,43 +436,6 @@
...
-    "\n",
-    "nncf_config_dict = {\n",
-    "    \"input_info\": {\"sample_size\": [1, 1, 512, 512]},\n",
-    "    \"target_device\": \"CPU\",\n",
-    "    \"compression\": {\n",
-    "        \"algorithm\": \"quantization\",\n",
-    "        # performance preset uses symmetric weights and activations\n",
-    "        \"preset\": \"performance\",\n",
-    "        # Do not quantize LeakyReLU activations to allow the INT8 model to run on Intel GPU\n",
-    "        \"ignored_scopes\": [\"{re}.*LeakyReLU*\"],\n",
```
Comment (eaidova) [inline notebooks/110-ct-segmentation-quantize/110-ct-segmentation-quantize-nncf.ipynb:463]: Are you sure that configuration is not actual anymore?
 At least ignored scope, could you please test int8 model inference on iGPU?

File: notebooks/110-ct-segmentation-quantize/110-ct-segmentation-quantize-nncf.ipynb [hunk 2]
```diff
@@ -481,7 +450,7 @@
     "tags": []
    },
    "source": [
-    "Create a quantized model from a pre-trained `FP32` model and configuration object created with the dictionary defined in the previous cell and a DataLoader. See the [NNCF documentation](https://github.com/openvinotoolkit/nncf/blob/develop/docs/Usage.md) for more information."
+    "Create a quantized model from the pre-trained `FP32` model and the calibration dataset."
```
Comment (eaidova) [inline notebooks/110-ct-segmentation-quantize/110-ct-segmentation-quantize-nncf.ipynb:453]: could you please provide more explanation for this part?

Comment (aleksandr-mokrov) [reply notebooks/110-ct-segmentation-quantize/110-ct-segmentation-quantize-nncf.ipynb]: More description was added.

File: notebooks/110-ct-segmentation-quantize/110-ct-segmentation-quantize-nncf.ipynb (hunk 1 above)
Comment (aleksandr-mokrov) [reply notebooks/110-ct-segmentation-quantize/110-ct-segmentation-quantize-nncf.ipynb:463]: You are right, it doesn't work on iGPU. I returned this configuration settings.

File: notebooks/110-ct-segmentation-quantize/data-preparation-ct-scan.ipynb [hunk 3]
```diff
@@ -28,7 +28,7 @@
     "\n",
```
Comment (eaidova) [inline notebooks/110-ct-segmentation-quantize/data-preparation-ct-scan.ipynb:28]: <pre class="ql-syntax" spellcheck="false">Line #2.    NIFTI_PATH = Path("~/PycharmProjects/default/kits19/data").expanduser()
</pre><p>please restore path</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/965/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

**Comment by eaidova** [comment]
@aleksandr-mokrov you have merge conflict, could you please resolve it?

**Comment by eaidova** [comment]
```
 notebooks/110-ct-segmentation-quantize/110-ct-segmentation-quantize-nncf.ipynb:cell_3:38:5: F401 'nncf.NNCFConfig' imported but unused
notebooks/110-ct-segmentation-quantize/110-ct-segmentation-quantize-nncf.ipynb:cell_3:39:5: F401 'nncf.torch.create_compressed_model' imported but unused
notebooks/110-ct-segmentation-quantize/110-ct-segmentation-quantize-nncf.ipynb:cell_3:39:5: F401 'nncf.torch.register_default_init_args' imported but unused
```
@aleksandr-mokrov please remove unused imports
